src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -249,7 +249,7 @@ public static int getSkyBlendColour(World world, BlockPos center)
             for (int z = -distance; z <= distance; ++z)
             {
                 BlockPos pos = center.add(x, 0, z);
-                Biome biome = world.getBiomeGenForCoords(pos);
+                Biome biome = world.getBiome(pos);
```

Comment:
This change looks unrelated to fluids? Seems like it belongs on a separate PR ;P